src/grisp_tools_io.erl

```diff
@@ -79,6 +79,16 @@ get(latin1, Data) ->
         false ->
             no_clue
     end;
+get(trim_string, []) ->
+    no_data;
+get(trim_string, String) ->
+    case is_list(String) of
+        true ->
+            Trimmed = string:trim(String),
+            unicode:characters_to_binary(Trimmed);
```

Comment:
Just as a note, but is not needed.
This does not protect from `"` that if left unescaped will lead to crashes in the config file.
And if the token is between `""` will also crash  -> `<<""mytoken"">>`